src/services/utilities.ts

```diff
@@ -1701,12 +1701,26 @@ namespace ts {
         return !!compilerOptions.module || compilerOptions.target! >= ScriptTarget.ES2015 || !!compilerOptions.noEmit;
     }
 
-    export function hostUsesCaseSensitiveFileNames(host: { useCaseSensitiveFileNames?(): boolean; }): boolean {
-        return host.useCaseSensitiveFileNames ? host.useCaseSensitiveFileNames() : false;
+    export function createModuleSpecifierResolutionHost(program: Program, host: LanguageServiceHost): ModuleSpecifierResolutionHost {
+        // Mix in `getProbablySymlinks` from Program when host doesn't have it

```

Comment:
```suggestion
        // Mix in `getProbableSymlinks` from Program when host doesn't have it
```

I think this was originally my typo but I just noticed it 😄